src/main/java/com/microsoft/azure/functions/annotation/FixedDelayRetry.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.functions.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * <p>
+ * Defines a retry strategy where a fixed delay is used between retries.</p>
+ *
+ * @since 1.0.0
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.PARAMETER)
```

Comment:
The target should be Method correct?

Reply:
Thank you for the review, @amamounelsayed . In this case, which ElementType to use? ElementType.Constructor? The original C# Implementation looks like this. 

```csharp
//        [FixedDelayRetry(3, "00:00:3")]
        [ExponentialBackoffRetry(3, "00:00:05", "00:00:30")]
        [FunctionName(nameof(SampleConsumerAsync))]
        public async Task SampleConsumerAsync(
```

Reply:
Well to support the annotation to be function level, the @Target(ElementType.METHOD)
As @pragnagopa point, I believe it should be too @Target(ElementType.Type). We may need to check it.

Reply:
Thanks. I update it as Method.